glados-web/templates/contentaudit_detail.html

```diff
@@ -14,11 +14,13 @@
             <ul>
                 <li>Content Key: <a href="/content/key/{{content.key_as_hex() }}/">{{ content.key_as_hex() }}</a></li>
                 <li>Content ID: <a href="/content/id/{{content.id_as_hex() }}/">{{ content.id_as_hex() }}</a></li>
+                {% if execution_metadata.is_some() %}
                 <li>Block:
-                    <a href="https://etherscan.io/block/{{ execution_metadata.block_number }}">
-                        {{ execution_metadata.block_number }}
+                    <a href="https://etherscan.io/block/{{ execution_metadata.as_ref().unwrap().block_number }}">
+                        {{ execution_metadata.as_ref().unwrap().block_number }}
```

Comment:
The value is getting a little unwieldy. Maybe generate it once and reuse, like:
```suggestion
                    {% let block_num = execution_metadata.as_ref().unwrap().block_number %}
                    <a href="https://etherscan.io/block/{{ block_num }}">
                        {{ block_num }}
```
[source docs](https://djc.github.io/askama/template_syntax.html#assignments)